src/main/java/xyz/wagyourtail/jsmacros/client/api/helpers/ClientPlayerEntityHelper.java

```diff
@@ -56,6 +57,14 @@ public ClientPlayerEntityHelper(T e) {
     public int getFoodLevel() {
         return base.getHungerManager().getFoodLevel();
     }
+
+    /**
+     * @since 1.4.x
+     * @return the {@code input} values of the entity.
+     */
+    public Input getInput() {
```

Comment:
wrap this with an `InputHelper` class (don't pass raw minecraft classes to the user)

Reply:
I only used this in FPlayer getCurrentInput() in the past, so I think, I will remove this.